.goreleaser_client.yaml

```diff
@@ -0,0 +1,25 @@
+project_name: tuf-client
+builds:
+  - ldflags:
+      - "-s -w"
+      - "-extldflags=-zrelro"
+      - "-extldflags=-znow"
+    env:
+      - "CGO_ENABLED=0"
+      - "GO111MODULE=on"
+      - "GOFLAGS=-mod=readonly -trimpath"
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+    main: ./cmd/tuf-client/
+sboms:
+  - artifacts: archive
+signs:
+  - cmd: cosign
```

Comment:
maybe we can move the COSIGN_EXPERIMENTAL environment variable over here under the env property of the sign section, WDYT?

Reply:
You mean in the goreleaser files instead of the github action workflow file? Do you know how? Can you provide a gist?

Reply:
of course, let me clarify this example:

https://github.com/goreleaser/goreleaser/blob/bd33dc37ec450adde42e139a147e45e7a7ffde0f/.goreleaser.yaml#L209